internal/commands/vaultsecrets/secrets/update_test.go

```diff
@@ -164,6 +168,24 @@ details = {
 	  "collection_name" = "cn2"
 	}]
   }
+}`),
+		},
+		{
+			Name:    "Success: Update a Postgres rotating secret",
+			RespErr: false,
+			AugmentOpts: func(o *UpdateOpts) {
+				o.Type = secretTypeRotating
+			},
+			MockCalled: true,
+			Provider:   integrations.Postgres,
+			Input: []byte(`type = "postgres"
+details = {
+  rotate_on_update = true
+  integration_name = "postgres-integration"
+  rotation_policy_name = "built-in:60-days-2-active"
+  postgres_params = {
+  	usernames = ["postgres_user_1"]
```

Comment:
```suggestion
  	usernames = ["postgres_user_1", "postgres_user_2"]
```

Might be better to test with two users since that's what the server expects.